nss-day-cohort-39/nutshell-heliotrope-honey-badgers
# Ac messages

Steps to test:
1. login as a user
2. check to see that session storage updated to show the correct active user
3. send a message
4. upon sending, the message list should re-render automatically, and automatically scroll to the bottom of the message field (assuming there's overflow)
5. refresh the page and login as another user
6. repeat steps 2-5
